internal/provider/service_resource.go

```diff
@@ -1148,14 +1155,13 @@ func (r *ServiceResource) ModifyPlan(ctx context.Context, req resource.ModifyPla
 					"Use: io1 for volume_type if volume_iops is set")
 			return
 		}
-		if !plan.VolumeIOPS.IsNull() &&
-			plan.VolumeType.ValueString() != "io1" {
+		if !plan.VolumeIOPS.IsNull() && plan.VolumeType.ValueString() != "io1" {
```

Comment:
Don't we allow setting IOPS with `gp3` as well?

Reply:
thanks for catching this, will update